code/zoltar_scripts/README.md

```diff
@@ -0,0 +1,36 @@
+We use Zoltar to store forecasts and metadata. This means all forecasts and metadata can be accessed via web download or programatically using R or Python.
+
+- [View the ECDC European COVID-19 Forecast Hub on Zoltar](https://www.zoltardata.com/project/238)
+- [Find out more about Zoltar](https://docs.zoltardata.com/)
+
+----
+
+### Hub developers
+
+#### Local upload to Zoltar
+
+First ensure you have set up  hub Zoltar access by adding `Z_USERNAME` and `Z_PASSWORD` as environment variables.
+
+```
+# Clone repo with validations submodule
+git clone epiforecasts/covid19-forecast-hub-europe --recurse-submodules
+
+# Set hub as working directory, e.g.
+# cd Github/covid19-forecast-hub-europe
```

Comment:
minimally minor comment but would just make it `cd covid19-forecast-hub-europe` as everything else depends on local structure.